### DISCUSSION TITLE
Adding new pixels around existing pixels on a layer automatically

### DISCUSSION THREAD

**Discussion by yippiekae** [sn4k3/UVtools#770] [Q&A] [answered]
Hi,
I've been tinkering with the great UVtools for a while, especially in the pixel arithmetic section, but can't find what I need :
I want to add pixels around existing pixels on certain layer. For example, if a there is a layer with just 1 white pixel, I want to "light up" all the 8 surrounding pixels.
Is there a way to do that ?

Thank you ! Flo

**Comment by sn4k3** [comment, marked as answer]
You can use Tool - Morph with dilate operation, it will expand all areas by the desired amount. Select just the layer to process.

**Reply by yippiekae** [reply]
Thank you very much, that's exactly what I was looking for !
